Remotion/Web/Core/Res/Themes/NovaViso/HTML/TreeView.css

```diff
@@ -89,16 +89,41 @@ ul.treeViewRoot span.treeViewNode > a img.Icon
   margin-right: var(--treeview-chevron-gap);
 }
 
-ul.treeViewRoot span.treeViewNode
+ul.treeViewRoot span.treeViewNode::before
 {
+  content: "";
   padding-left: calc(var(--treeview-padding) * var(--nesting-depth));
 }
 
-ul.treeViewRoot ul.treeViewTopLevelNodeChildren span.treeViewNode
+ul.treeViewRoot ul.treeViewTopLevelNodeChildren span.treeViewNode::before
 {
+  content: "";
   padding-left: calc(var(--treeview-padding) * (var(--nesting-depth) - 1));
 }
 
+ul.treeViewRoot span.treeViewNode span.treeViewNodeBadge
+{
+  position: sticky;
+  right: 0;
+  background-color: var(--color-standard-background);
+  align-self: stretch;
+  margin-left: auto;
+  padding: 0 var(--remotion-themed-spacing);
```

Comment:
I tweaked this, the screenshot test will be red

Reply:
What is the plan for the screenshot test because they will only work with one theme. Are we running the tests using new theme + no IE?

Reply:
right now: nova gray.
future: separate test runs for each theme (.net fx = novagray, net5 = novaviso)